python/pyspark/testing/connectutils.py

```diff
@@ -76,6 +76,7 @@ class MockRemoteSession:
     def __init__(self):
         self.hooks = {}
         self.session_id = str(uuid.uuid4())
+        self.is_stopped = True
```

Comment:
I think you can name it `self.is_mocksession`, and `if not hasattr(self._session, "is_mocksession"):` with commenting that it indicates `MockRemoteSession`

Reply:
let me switch to `is_mocksession`